code/process_sequences.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Delong Tsway
+# dttsway@dons.usfca.edu
+# 2020-10-18
+
+# resources I consulted to build this code:
+# https://github.com/vsbuffalo/bioawk-tutorial/blob/master/README.md
+# https://opensource.com/article/17/2/command-line-tools-data-analysis-linux
+# https://www.poftut.com/awk-text-split-delimit-examples/
+# https://www.cyberciti.biz/faq/run-execute-sh-shell-script/
+# https://askubuntu.com/questions/420981/how-do-i-save-terminal-output-to-a-file
+
+FILE="/blast-db/sars-cov-2-seq-data/2020-10-15_ncbi_sars_cov_2_sequences.fasta.gz"
+
+if [ $# -eq 0 ]
+then
+ echo "There is no file path provided"
+exit 1
+fi
+
+if [ $# -gt 1 ]
+then
+echo "There is more than the one require argument provided"
+exit 1
+fi
+# echo statement to describe contents:
+echo "This is some information about the sequences from the Sars-Cov-2 database" > output/SeqSort.txt
```

Comment:
Don't send these to a file, just print them to the screen, so that when you redirect the output of this script, it has the description in the same file and if you don't redirect the output, then the helpful information is printed to the screen.